src/pages/_app.js

```diff
@@ -17,6 +17,7 @@ function App({ Component, pageProps }) {
             items: [
               {
                 title: 'Fabric',
+                href: '/',
```

Comment:
Why is this linking to the home page? If it's because we dont yet have the pages for `principles`, `what's new `and `getting started` we can probably just remove this section for now imo.

Reply:
As mentioned in the comment above, the index page is not picked up by the site atm, or at least it was not earlier today, so Its temporarily linked to something to be able to work on the landing page at all.

Reply:
Removed the temporary link, seams to work without it now anyways :)